ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -685,7 +685,19 @@ function upgrade() {
         clear_upgrade
         check_upgrade_status "${cluster_src_ver}"
     fi
-    run_command "oc adm upgrade --to-image=${TARGET} --allow-explicit-upgrade --force=${FORCE_UPDATE}"
+
+    cmd="oc adm upgrade --to-image=${TARGET} --allow-explicit-upgrade"
+    if [[ "$TARGET_VERSION" != *"nightly"* ]]; then
+        conditionalUpdates=$(oc get clusterversion version -o json | jq -r '.status.conditionalUpdates[]?.version')
+        if [[ "$conditionalUpdates" == *"$TARGET_VERSION"* ]]; then
+            echo "Error: $TARGET_VERSION is not recommended, for details please refer:"
+            oc get clusterversion version -o json | jq -r '.status.conditionalUpdates'
+        else
+            run_command "$cmd"
+        fi
+    else
+        run_command "$cmd --force=${FORCE_UPDATE}"
```

Comment:
Actually we do not need to interfere `FORCE_UPDATE`, just keep the origin line, just add a pre-check, then exit before run this upgrade command line.

Reply:
Sorry, i don't understand you

Reply:
Something like:
```
<check if the target payload is not recommended, if yes, exit>
run_command "oc adm upgrade --to-image=${TARGET} --allow-explicit-upgrade --force=${FORCE_UPDATE}"
```